spec.html

```diff
@@ -39468,14 +39429,13 @@ <h1>GeneratorYield ( _iterNextObj_ )</h1>
     <h1>AsyncGenerator Objects</h1>
     <p>An AsyncGenerator object is an instance of an async generator function and conforms to both the AsyncIterator and AsyncIterable interfaces.</p>
 
-    <p>AsyncGenerator instances directly inherit properties from the object that is the value of the *"prototype"* property of the AsyncGenerator function that created the instance. AsyncGenerator instances indirectly inherit properties from the AsyncGenerator Prototype intrinsic, %AsyncGenerator.prototype%.</p>
+    <p>AsyncGenerator instances directly inherit properties from the object that is the initial value of the *"prototype"* property of the AsyncGenerator function that created the instance. AsyncGenerator instances indirectly inherit properties from the AsyncGenerator Prototype intrinsic, %AsyncGeneratorFunction.prototype.prototype%.</p>
 
     <emu-clause id="sec-properties-of-asyncgenerator-prototype">
       <h1>Properties of the AsyncGenerator Prototype Object</h1>
       <p>The AsyncGenerator prototype object:</p>
       <ul>
-        <li>is <dfn>%AsyncGeneratorPrototype%</dfn>.</li>
-        <li>is <dfn>%AsyncGenerator.prototype%</dfn> (<dfn>%AsyncGeneratorFunction.prototype.prototype%</dfn>).</li>
+        <li>is the initial value of the *"prototype"* property of %AsyncGeneratorFunction.prototype%.</li>
```

Comment:
```suggestion
        <li>is <dfn>%AsyncGeneratorFunction.prototype.prototype%</dfn>.</li>
```

Reply:
This isn't needed, because it's not a top-level intrinsic (i'll still delete the "initial value" line, tho, thanks!)